scripts/build.sh

```diff
@@ -15,9 +10,20 @@ LDFLAGS=(
    -X main.BuildDate="$(date -u '+%Y-%m-%dT%I:%M:%S%p')"
 )
 
-VERSION=$(go run cmd/scw/main.go -o json version | jq -r .version | tr . -)
-
 export CGO_ENABLED=0
-GOOS=linux  GOARCH=amd64 go build -ldflags "${LDFLAGS[*]}" -o "$BIN_DIR/scw-$VERSION-linux-x86_64"  cmd/scw/main.go
-GOOS=darwin GOARCH=amd64 go build -ldflags "${LDFLAGS[*]}" -o "$BIN_DIR/scw-$VERSION-darwin-x86_64" cmd/scw/main.go
-GOOS=windows GOARCH=amd64 go build -ldflags "${LDFLAGS[*]}" -o "$BIN_DIR/scw-$VERSION-windows-x86_64.exe" cmd/scw/main.go
+
+if [[ "${1}" == "build-in-docker" ]]; then
```

Comment:
what happens if 0 arg is passed ? does this fail ? 

isn't it better to have a flag that we set to `true` ?

Reply:
if 0 args are passed `$1` is an empty string